### PR TITLE
fix(plugin-meeting): upgrade webrtc adapter

### PR DESCRIPTION
**Pull request by arun3528**
Upgrading the webrtc adapter to support latest version of browser 

Make sure to have followed the [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request) before submitting.

### CODE REVIEW

**Review by mwegman — APPROVED** [review]

**Comment by adamweeks** [comment]
@arun3528 is this PR good to merge? I was told to wait on it for your approval.

**Comment by arun3528** [comment]
@adamweeks we can merge this PR

**Pull request merged**
